extension/js/common/composer.ts

```diff
@@ -1201,6 +1208,45 @@ export class Composer {
         if (method === 'forward') {
           this.urlParams.subject = 'Fwd: ' + this.urlParams.subject;
           await this.retrieveDecryptAddForwardedMsg(determined.lastMsgId);
+        } else {
+          try {
+            const { raw } = await Google.gmail.msgGet(this.urlParams.acctEmail, this.urlParams.threadMsgId, 'raw');
+            const msgMimeContent = await Mime.process(Buf.fromBase64UrlStr(raw!));
+            if (!msgMimeContent.from || !msgMimeContent.headers.date || !msgMimeContent.blocks || !msgMimeContent.blocks.length) {
+              return;
+            }
+            if (!this.urlParams.draftId) {
+              const blocksToDisplay = msgMimeContent.blocks.filter(b => b.type === 'encryptedMsg' ||
+                b.type === 'plainText' || b.type === 'plainHtml');
+              const contentToDisplay: Array<string> = [];
+              for (const block of blocksToDisplay) {
+                if (block.type === 'encryptedMsg') {
+                  const armored = Pgp.armor.clip(block.content as string);
+                  if (armored) {
+                    const decrypted = await PgpMsg.decrypt({
+                      kisWithPp: await Store.keysGetAllWithPassphrases(this.urlParams.acctEmail),
+                      encryptedData: Buf.fromUtfStr(armored)
+                    });
+                    if (decrypted.success) {
+                      contentToDisplay.push(decrypted.content.toUtfStr());
```

Comment:
```js
} else {
  contentToDisplay.push(`\n(Failed to decrypt quote from previous message because: ${decrypted.error.type}: ${decrypted.error.message})\n`);
}
```
or something like that. Easier to debug issues